common/src/main/java/com/ably/tracking/common/Ably.kt

```
@@ -86,6 +86,21 @@ interface Ably {
         callback: (Result<Unit>) -> Unit
     )
 
+    /**
+     * Sends a raw location update to the channel.
+     * Should be called only when there's an existing channel for the [trackableId].
+     * If a channel for the [trackableId] doesn't exist then it just calls [callback] with success.
```

Do you rely on this behaviour? This feels surprising to me, at least when examining this API in isolation. I generally feel it's better to prefer failure (a.k.a. fail early) rather than 'hide' unexpected cases. 🤔

---

I don't think that we use this behaviour but since it is how all methods in this interface work I would prefer to create a separate issue to change this instead of changing it in this PR :wink: WDYT? Should I create the issue?

---

Yes. Happy to see an issue created for this, under [code-quality](https://github.com/ably/ably-asset-tracking-android/labels/code-quality).

---

Issue created: https://github.com/ably/ably-asset-tracking-android/issues/496